e2e/api/backend_test.go

```diff
@@ -262,12 +262,12 @@ func (s *APIBackendTestSuite) TestRestartNode() {
 	require := s.Require()
 	require.NotNil(s.Backend)
 
-	s.StartTestBackend(params.RinkebyNetworkID)
+	s.StartTestBackend()
 	defer s.StopTestBackend()
 
 	firstHash, err := e2e.FirstBlockHash(s.Backend.NodeManager())
 	s.NoError(err)
-	s.Equal("0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177", firstHash)
+	s.Equal("0x28c4da1cca48d0107ea5ea29a40ac15fca86899c52d02309fa12ea39b86d219c", firstHash)
```

Comment:
It's going to be different for each network.

Reply:
Then should we make it test specifically for `statuschain` network or have a map with associated `chain` id for each network?

Reply:
Why could we potentially want to run this test only on StatusChain?